tests/integration/spark/test_reducer_merge.py

```diff
@@ -196,3 +197,27 @@ def test_distributed_asnumpy(self):
         float_32_dtype = numpy.dtype("float32")
         self.assertEqual(npy_x.dtype, int_32_dtype)
         self.assertEqual(npy_y.dtype, float_32_dtype)
+
+    def test_distributed_snapshot(self):
+        """Test support for `Snapshot` in distributed backend"""
+        # A simple dataframe with ten sequential numbers from 0 to 9
+        df = PyRDF.RDataFrame(10).Define("x", "(int)rdfentry_")
+
+        # Count rows in the dataframe
+        nrows = df.Count()
+
+        # Snapshot to two files, build a ROOT.TChain with them and retrieve a
+        # PyRDF.RDataFrame
+        snapdf = df.Snapshot("snapTree", "snapFile.root")
+
+        # Count the rows in the snapshotted dataframe
+        snapcount = snapdf.Count()
+
+        self.assertEqual(nrows.GetValue(), 10)
+        self.assertEqual(snapcount.GetValue(), 10)
+
```

Comment:
Here we could check that the corresponding intermediate root files have been created with the expected name before removing them later (`os.path.exists("snapFile_X_Y.root")`, ...)

Reply:
ok!